### PR TITLE
RNs-0.1.32 Added with Bug Fix

### PR DESCRIPTION
**Pull request by Wiharris**
**[OSDOCS-4400](https://issues.redhat.com//browse/OSDOCS-4400)** 
RNs-0.1.31 Added with Bug Fix

**Version(s):**
PR applies to all versions after **4.8+**

**Issue:**
https://issues.redhat.com/browse/OSDOCS-4400

**Link to docs preview:**
https://52110--docspreview.netlify.app/openshift-enterprise/latest/security/file_integrity_operator/file-integrity-operator-release-notes.html#file-integrity-operator-release-notes-0-1-31

QE review:
- [ ] QE has approved this change.

**Additional information:**
`shipped live`

### CODE REVIEW

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 1]
```diff
@@ -13,6 +13,28 @@ These release notes track the development of the File Integrity Operator in the
...
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-0-1-31"]
+== OpenShift File Integrity Operator 0.1.31
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.31:
+
+* link:https://access.redhat.com/errata/RHBA-2022:103760[RHBA-2022:103760 OpenShift File Integrity Operator Bug Fix and Enhancement Update]
+
+[id="file-integrity-operator-0-1-31-new-features-and-enhancements"]
```
Comment (sheriff-rh) [inline security/file_integrity_operator/file-integrity-operator-release-notes.adoc:23]: This features/enhancement section  is not needed as it is documented in 0.1.30.

**Review by sheriff-rh — COMMENTED** [review]
Looks good! Couple comments.

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 2]
```diff
@@ -13,6 +13,28 @@ These release notes track the development of the File Integrity Operator in the
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-0-1-31"]
+== OpenShift File Integrity Operator 0.1.31
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.31:
+
+* link:https://access.redhat.com/errata/RHBA-2022:103760[RHBA-2022:103760 OpenShift File Integrity Operator Bug Fix and Enhancement Update]
```
[the remark this answers is not in the document]
Comment (Wiharris) [reply security/file_integrity_operator/file-integrity-operator-release-notes.adoc]: Done! Thanks for your help!

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc (hunk 1 above)
Comment (Wiharris) [reply security/file_integrity_operator/file-integrity-operator-release-notes.adoc]: Done! :)

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 3]
```diff
@@ -13,6 +13,20 @@ These release notes track the development of the File Integrity Operator in the
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-0-1-31"]
+== OpenShift File Integrity Operator 0.1.31
```
Comment (rhmdnd) [inline security/file_integrity_operator/file-integrity-operator-release-notes.adoc:17]: We hit a regression with 0.1.31 on 4.6. We already landed a fix with https://github.com/openshift/file-integrity-operator/pull/291 and we're in the process of creating 0.1.32:

https://github.com/openshift/file-integrity-operator/pull/294

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 4]
```diff
@@ -13,6 +13,20 @@ These release notes track the development of the File Integrity Operator in the
...
+The following advisory is available for the OpenShift File Integrity Operator 0.1.31:
+
+* link:https://access.redhat.com/errata/RHBA-2022:7095[RHBA-2022:7095 OpenShift File Integrity Operator Bug Fix Update]
+
+[id="file-integrity-operator-0-1-31-bug-fixes"]
+=== Bug fixes
+
+* Previously, alerts issued by the File Integrity Operator did not set a namespace, making it difficult to understand where the alert originated. Now, the Operator sets the appropriate namespace, increasing understanding of the alert. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2112394[*BZ#2112394*])
+
+* Previously, The File Integrity Operator did not update the metrics service on operator startup, causing the namespace to be unreachable for long periods. With this release, the File Integrity Operator now ensures the metrics service is updated on operator startup. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2115821[*BZ#2115821*])
```
Comment (mrogers950) [inline security/file_integrity_operator/file-integrity-operator-release-notes.adoc:28]: `causing the namespace to be unreachable for long periods.` 
We should say that it's the metrics targets that are unreachable.

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc (hunk 3 above)
Comment (Wiharris) [reply security/file_integrity_operator/file-integrity-operator-release-notes.adoc]: Thanks for the review and feedback! I have made the appropriate changes. :)

**Review by rhmdnd — APPROVED** [review]
/lgtm

I'm good with these changes. Thanks @Wiharris!

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 5]
```diff
@@ -13,6 +13,20 @@ These release notes track the development of the File Integrity Operator in the
...
+== OpenShift File Integrity Operator 0.1.32
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.32:
+
+* link:https://access.redhat.com/errata/RHBA-2022:7095[RHBA-2022:7095 OpenShift File Integrity Operator Bug Fix Update]
+
+[id="file-integrity-operator-0-1-32-bug-fixes"]
+=== Bug fixes
+
+* Previously, alerts issued by the File Integrity Operator did not set a namespace, making it difficult to understand where the alert originated. Now, the Operator sets the appropriate namespace, increasing understanding of the alert. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2112394[*BZ#2112394*])
```
Comment (kcarmichael08) [inline security/file_integrity_operator/file-integrity-operator-release-notes.adoc:26]: Suggestion: "...making it difficult to understand from which namespace the alert originated."

Comment (kcarmichael08) [reply security/file_integrity_operator/file-integrity-operator-release-notes.adoc:26]: "increasing understanding of the alert" - this is a little awkward, maybe "providing more information about the alert" or similar?

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 6]
```diff
@@ -13,6 +13,20 @@ These release notes track the development of the File Integrity Operator in the
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-0-1-32"]
+== OpenShift File Integrity Operator 0.1.32
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.32:
+
+* link:https://access.redhat.com/errata/RHBA-2022:7095[RHBA-2022:7095 OpenShift File Integrity Operator Bug Fix Update]
```
Comment (kcarmichael08) [inline security/file_integrity_operator/file-integrity-operator-release-notes.adoc:21]: Link not working yet

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 7]
```diff
@@ -13,6 +13,20 @@ These release notes track the development of the File Integrity Operator in the
...
+
+[id="file-integrity-operator-0-1-32-bug-fixes"]
+=== Bug fixes
+
+* Previously, alerts issued by the File Integrity Operator did not set a namespace, making it difficult to understand where the alert originated. Now, the Operator sets the appropriate namespace, increasing understanding of the alert. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2112394[*BZ#2112394*])
+
+* Previously, The File Integrity Operator did not update the metrics service on operator startup, causing the metrics targets to be unreachable. With this release, the File Integrity Operator now ensures the metrics service is updated on operator startup. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2115821[*BZ#2115821*])
```
Comment (kcarmichael08) [inline security/file_integrity_operator/file-integrity-operator-release-notes.adoc:28]: I think Operator needs to be capitalized in "updated on Operator startup" (it's capitalized in the previous paragraph). The SSG seems to imply it's always capitalized.

**Review by kcarmichael08 — COMMENTED** [review]
Link not active yet so couldn't check, couple of suggestions but otherwise LGTM!

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc (hunk 5 above)
Comment (Wiharris) [reply security/file_integrity_operator/file-integrity-operator-release-notes.adoc]: @kcarmichael08 Thanks for the suggestion! Updates have been made. :)

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc (hunk 6 above)
Comment (Wiharris) [reply security/file_integrity_operator/file-integrity-operator-release-notes.adoc:21]: Links will not show until errata is `shipped live`.

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc (hunk 7 above)
Comment (Wiharris) [reply security/file_integrity_operator/file-integrity-operator-release-notes.adoc]: Thanks for the catch! Done! :)

**Pull request merged**